standard/arrays.md

```diff
@@ -174,7 +174,7 @@ The context in which an array initializer is used determines the type of the arr
 > ```
 > *end example*
 
-> it is simply shorthand for an equivalent array creation expression:
+It is simply shorthand for an equivalent array creation expression:
```

Comment:
@RexJaeschke  

Can you look at this edit and suggest any needed changes?

It was a single sentence (fragment) in a block quote, but not included in either of the two examples. It doesn't seem to make sense that this is a normative statement, but it's currently not in a *note* or *example*.

What's your suggestion?

Reply:
Bill, I believe all this should be normative, and I just made changes to do that.